matrix/MyMatrix.py

```diff
@@ -1,22 +1,22 @@
 class MyMatrix:
-    matrix = [[]]
-
-    def __init__(self, matrix):
-        self.matrix = matrix
+    def __init__(self, a, b, x):
```

Comment:
Nie rozumiem, czym jest a, czym jest b, czym jest x?

Reply:
zad2.pdf:
"Dla losowej macierzy kwadratowej A i wektora B, rozwiąż układ równań liniowych A*X=B"
A - macierz
X - wektor
B = A * X

Reply:
da radę do jakoś ładniej nazwać niż A, B i X?

Reply:
Well yes, nazwałam je po prostu tak, jak było w poleceniu :v "a" to może być po prostu matrix, a "b" i "x"? vectorb/vectorx czy masz lepszy pomysł?